test/e2etest/installer_test.go

```diff
@@ -23,7 +23,9 @@ func TestInstaller(t *testing.T) {
 	runInstallScript(testConfig.ConfigPath)
 
 	// run again with same config. This is to assure that the `re-apply` action finishes without any issue.
-	runInstallScript(testConfig.ConfigPath)
+	if !useDomain {
+		runInstallScript(testConfig.ConfigPath)
```

Comment:
Disable it for the time being, because there is build failure caused by it. 
More detail can be found in [CLIP-1673](https://bulldog.internal.atlassian.com/browse/CLIP-1673).